scenes/game/local_game.gd

```diff
@@ -2171,7 +2279,7 @@ func handle_strike_effect(card_id :int, effect, performing_player : Player):
 		"powerup_damagetaken":
 			var power_per_damage = effect['amount']
 			var total_powerup = power_per_damage * active_strike.get_damage_taken(performing_player)
-			if total_powerup > 0:
+			if total_powerup != 0 and (total_powerup > 0) == (power_per_damage > 0):
```

Comment:
Can you rewrite this to not be a somewhat confusing if case?
It's okay to have more lines/if blocks, I just prefer if it is readable on sight as to what is going on.
I favor explicit behavior at a glance over brevity for stuff like this.